lib/roadie/style_block.rb

```diff
@@ -36,7 +36,8 @@ def inlinable?
     # used in the DOM.
     # @return {String}
     def to_s
-      "#{selector}{#{properties.map(&:to_s).join(';')}}"
+      rules = properties.map { |rule| "#{rule.to_s};" }.join('')
+      "#{selector}{#{rules}}"
```

Comment:
I preferred the previous version where redundant semicolons were removed. Is there any reason why you changed this?